### ISSUE TITLE
IKEA E1812 Deconz not working

### ISSUE THREAD

**Issue by eloo** [EPMatt/awesome-ha-blueprints#42]
Thank you for taking the time to report here! :tada:

**Please don't delete any part of the template**, since keeping the provided structure will help maintainers to process your request more rapidly.

This template should only be used to report a bug with a blueprint. If this is not your case, you can [choose a different issue template](https://github.com/EPMatt/awesome-ha-blueprints/issues/new/choose). If you have a question instead, please open a discussion [here](https://github.com/EPMatt/awesome-ha-blueprints/discussions/new).

\*_=required_

## Bug Details\*

- Blueprint name: EPMatt/ikea_e1812.yaml
- Home Assistant Core Version: 2021.4.6
- Home Assistant Installation Type: Home Assistant Supervised

## Description\*

I have configured the controller for the e1812 with some simple actions (running a script) but it seems that the controller is not executing the configured actions.

## Automation YAML config\*

```yaml
alias: '[IKEA-Shortcut] Controller - Bertha Wohnzimmer saugen'
description: Lass Bertha das Wohnzimmer saugen
use_blueprint:
  path: EPMatt/ikea_e1812.yaml
  input:
    integration: deCONZ
    controller_device: 5c2adad9f497010a5d1cc7244401f1c8
    action_button_short:
      - service: script.bertha_wohnzimmer_saugen
      - condition: device
        device_id: cf31adae6946b68dc5796a8cfdf8606b
        domain: vacuum
        entity_id: vacuum.bertha
        type: is_cleaning
      - service: vacuum.pause
        target:
          entity_id: vacuum.bertha
    action_button_long:
      - service: script.bertha_stop
    action_button_double:
      - device_id: cf31adae6946b68dc5796a8cfdf8606b
        domain: vacuum
        entity_id: vacuum.bertha
        type: dock
    button_double_press: false
```

## To Reproduce\*

Steps to reproduce the issue:

1. Create a new automation from the blueprint
2. Add some actions
3. Trigger the automation
4. See that nothing happens

## Expected behavior\*

I would expect that the configured actions are running

## Actual Behaviour\*

Sscipts are not executed

## Additional Details\*

Calling the scripts without the automation is working as expected

## Screenshots

![image](https://user-images.githubusercontent.com/2631895/115448670-3d866780-a21a-11eb-92a9-6900fd0f09af.png)

**Comment by eloo** [comment]
Just figured out that it seems that the problem is coming from the added single quotes in the change from yesterday.
I have manually removed them and then its working. 
Maybe its because now a string is compared with an integer?

**Comment by EPMatt** [comment]
Hi @eloo, thank you for reporting and for your tests.
> Maybe its because now a string is compared with an integer?

Exactly. I suspect the problem is that the `deconz_event` is providing an integer, while we have defined deCONZ events as strings.

That change was made during an attempt of fixing an issue with double press events reported last week by a user on the HA Forum, which in turn was caused by a comparison between string and integer values.

I'll look into this and provide an updated blueprint which addresses this issue as well.
I'm sorry for any inconvenience this problem might have caused.
